_cite/util.py

```diff
@@ -95,6 +95,8 @@ def format_date(date):
 
     if isinstance(date, int):
         return datetime.fromtimestamp(date // 1000.0).strftime("%Y-%m-%d")
+    if isinstance(date, date_type):
```

Comment:
Do you intend to also allow `datetime`? If so, this should probably be

```suggestion
    if isinstance(date, date_type) or isinstance(date, datetime):
```

Reply:
This if branch should only occur when the user is manually specifying a date in the yaml and the yaml parser is parsing it. I think the only way it could be a datetime here is if the user provided a full iso datetime string in the yaml. Not sure why they'd do this since the template doesn't display times, but I've added it in for safety.